digiwf-engine/digiwf-engine-service/src/main/java/de/muenchen/oss/digiwf/process/definition/process/ProcessStartListener.java

```diff
@@ -0,0 +1,47 @@
+package de.muenchen.oss.digiwf.process.definition.process;
+
+import de.muenchen.oss.digiwf.process.definition.domain.service.ServiceInstanceInitializationService;
+import lombok.RequiredArgsConstructor;
+import org.camunda.bpm.engine.RepositoryService;
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class ProcessStartListener {
+
+    private final RepositoryService repositoryService;
+    private final ServiceInstanceInitializationService initializationService;
+
+    @EventListener(condition = "#execution.eventName.equals('start')")
+    public void startProcessInstance(DelegateExecution execution) {
+        if (isModelElementOfType("startEvent", execution)
+                && isProcessStartEvent(execution)
```

Comment:
Der Methoden-Name passt nicht ganz

Reply:
Aus meiner Sicht passt der Name - die Methode wird im Grunde (durch zusätzliche Einschänkung in Zeile 19 & 20) nur aufgerufen, wenn eine neue Execution am Prozessstart gestartet wurde (Execution start = Process instance start)

Reply:
Mit der Erläuterung [oben](https://github.com/it-at-m/digiwf-core/pull/1406#discussion_r1524459968), macht es Sinn